src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/MergeDataToSearchResultOperation.java

```diff
@@ -36,19 +37,24 @@ public SearchResultView execute(final CaseType caseType, final List<CaseDetails>
              ))
             .toArray(SearchResultViewColumn[]::new);
 
-        final SearchResultViewItem[] viewItems = caseDetails.stream()
-            .map(caseData -> new SearchResultViewItem(caseData.getReference().toString(), caseData.getData()))
-            .toArray(SearchResultViewItem[]::new);
+        final SearchResultViewItem[]
+            viewItems =
+            caseDetails.stream()
+                       .map(caseData -> new SearchResultViewItem(caseData.getReference().toString(),
+                                                                 caseData.getData(),
+                                                                 caseType.getCaseFields()
+                                                                         .stream()
+                                                                         .filter(caseField -> LABEL_FIELD_TYPE.equals(
+                                                                             caseField.getFieldType().getId()))
+                                                                         .collect(Collectors.toList())))
```

Comment:
Similarly, there's no tests asserting this logic, coverage is inherited from other tests

Reply:
Added test coverage https://github.com/hmcts/ccd-data-store-api/pull/132/files#diff-4b18d40268e75140ca920fc5cc9dab36